plugins/janus_audiobridge.c

```diff
@@ -6120,7 +6120,11 @@ static void *janus_audiobridge_handler(void *data) {
 						if(recording_base) {
 							/* Use the filename and path we have been provided */
 							g_snprintf(filename, 255, "%s-audio", recording_base);
-							participant->arc = janus_recorder_create(NULL, "opus", filename);
+							if(audiobridge->record_dir == NULL) {
+								participant->arc = janus_recorder_create(NULL, "opus", filename);
+							} else {
+								participant->arc = janus_recorder_create(audiobridge->record_dir, "opus", filename);
+							}
```

Comment:
There's no need for the if: just always pass `audiobridge->record_dir`, which if `NULL` means no dir.